### PR TITLE
Prepare a 2.0.6 release.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# vob 2.0.6 (2021-01-25)
+
+* Add `get_storage`.
```
Comment (thomwiggers) [inline CHANGES.md:3]:
```suggestion
* Add `get_storage`, available when `unsafe_internals` is enabled.
```

**Review by thomwiggers — COMMENTED** [review]
Great timing, I was just looking to fix the number of Github references in my `Cargo.toml`s :)

File: CHANGES.md (hunk 1 above)
Comment (ltratt) [reply CHANGES.md]: Good point. Fixed in https://github.com/softdevteam/vob/pull/55/commits/9a964ae74642c600b9a8c25bf1ddd4cea7d884a0. Probably best squashed before merging, assuming you're happy with my phrasing!

**Review by thomwiggers — APPROVED** [review]
Works for me!

**Comment by ltratt** [comment]
Squashed and ready for bors!

**Comment by ltratt** [comment]
@thomwiggers I think you need to reissue the command without the initial `@`.

**Comment by thomwiggers** [comment]
Seems like an oversight that bors would ignore `@bors`...

**Pull request merged**